### ISSUE TITLE
Missing optional dependency in documentation

### ISSUE THREAD

**Issue by Oakchris1955** [jimy-byerley/pymadcad#124]
Nowhere in the documentation is it mentioned that in order to be able to use most of the functionality in the `text` module one must also install `freetype-py` from PyPI.

**Comment by jimy-byerley** [comment]
You are right, we should declare it as an optional dependency in the setup and documentation :thinking:
